modules/core/src/lib/deck-renderer.js

```diff
@@ -46,16 +47,14 @@ export default class DeckRenderer {
 
     opts.layerFilter = this.layerFilter;
     opts.effects = opts.effects || [];
+    opts.target = opts.target || Framebuffer.getDefaultFramebuffer(this.gl);
```

Comment:
Nit: In the spirit of consistency / typing things out and making them self-documenting, should we call this `options.framebuffer` or `options.targetFramebuffer`?